libs/template/chunk/src/lib/chunk.directive.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  Directive,
+  Input,
+  NgZone,
+  OnDestroy,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core';
+import { RxStrategyProvider } from '@rx-angular/cdk/render-strategies';
+import { NextObserver, Subscription } from 'rxjs';
+
+/**
+ * @Directive ChunkDirective
+ *
+ * @description
+ *
+ * The `*rxChunk` directive serves as a convenient way for dividing template work into
+ * chunks. Applied to an element, it will schedule a task with the given RxRenderStrategy
+ * in order to postpone the template creation of this element.
+ *
+ * ### Features of `*rxChunk`
+ *
+ * - lightweight alternative to `*rxLet="[]"`
+ * - renderCallback
+ * - no value binding
+ * - no context variables
+ * - zone agnostic
+ * - suspense template
+ *
+ * @docsCategory ChunkDirective
+ * @docsPage ChunkDirective
+ * @publicApi
+ */
+@Directive({ selector: '[rxChunk]' })
+export class ChunkDirective implements OnInit, OnDestroy {
+  /**
+   * @description
+   * The rendering strategy with which the chunk directive should schedule the view
+   * creation. If no strategy is defined, `*rxChunk` will use the configured
+   * `primaryStrategy` as fallback instead.
+   *
+   * @example
+   * \@Component({
+   *   selector: 'app-root',
+   *   template: `
+   *     <div *rxChunk="strategy">
+   *       chunked template
+   *     </div>
+   *   `
+   * })
+   * export class AppComponent {
+   *   strategy = 'low';
+   * }
+   *
+   * @param { RxStrategyNames<string> } strategy
+   */
+  @Input('rxChunk') strategy = this.strategyProvider.primaryStrategy;
```

Comment:
Is it planned to work with not concurrent strategies?

Reply:
sure, but why would u? it basically makes the `rxChunk` do nothing useful anymore :D

Reply:
Yes, that's the thing. Can we type this input to RxConcurrentStrategyNames?

Reply:
We have to support all Strategien. Other strategies then concurrent will have an effect definitely. U could e.g. use 'noop' strategy when out of viewport

Reply:
There is a need to support all strategies. If you want to change the rendering strategy conditionally, then this is necessary. 

And it makes sense in many cases to switch the strategies based on conditions like first render and other things which may affect CLS